### DISCUSSION TITLE
NetBox Community Call - April 26th

### DISCUSSION THREAD

**Discussion by jeremystretch** [netbox-community/netbox#9135] [Announcements]
Our next NetBox community call  will be held on ~April 21st~ **Tuesday, April 26th at 4pm UTC (12pm EDT)**! We'll be discussing the recent v3.2 release, plugin development, and milestone selection for version 3.3 (due in August). Please join us!

If you have a question you'd like to ask, please [submit it here](https://forms.gle/NQR3Xzdom5Ztktp8A). We'll also be taking live questions from the chat as time allows.

You can bookmark the upcoming YouTube live stream here: https://youtu.be/StnCioePzmI

**Comment by candlerb** [comment]
FYI, the times shown are inconsistent: EDT is UTC-4, so 12pm EDT is 4pm UTC.

**Reply by jeremystretch** [reply]
Fixed, thanks!

**Comment by jeremystretch** [comment]
Apologies, folks! We've had to reschedule due to technical issues. We'll be holding this month's call on Tuesday, April 26th at 12pm EDT (same time as originally scheduled).

Be sure to note the new streaming link: https://youtu.be/StnCioePzmI
